nixos/modules/services/security/tor.nix

```diff
@@ -205,6 +212,7 @@ let
   optionFlags = lib.mkOption {
     type = with lib.types; listOf str;
     default = [ ];
+    internal = true;
```

Comment:
I'm not entirely sure about this one, from my reading of the module it looks like `flags` isn't supposed to be set by users directly, it's calculated from the per-flag attributes.